### DISCUSSION TITLE
Mehrere Hubs (mehrere homegear - Instanzen im Netz vorhanden) in HA Integration

### DISCUSSION THREAD

**Discussion by VirginEnt** [SukramJ/aiohomematic#1469] [Q&A]
Hallo zusammen,

das hier ist mein erster Post, lasst mich wissen, wenn ich gegen irgendwas verstoße oder groben Unsinn mache ;-) 

Zweitens: Besten Dank für die tolle Arbeit, diese Software zu entwickeln und der Community zur Verfügung zu stellen! Top. 

Hier meine Frage:
Sollte es eigentlich möglich sein, mehrere Hubs (in meinem Fall homegear - Instanzen) zur Integration hinzuzufügen? Jedenfalls suggeriert der Button "Hub hinzufügen" das irgendwie. Wenn ich dann das folgende Modal - Formular ausfülle und dann die Ports auswähle, kommt nur ein leerer (Bestätigungs-)Dialog, weiter passiert aber nix. 

Vielen Dank vorab für Eure Hilfe!
VG
Daniel

**Comment by Baxxy13** [comment]
Mit mehreren CCU's / RaspberryMatic's funktioniert das problemlos.
Zu Homegear kann ich leider nichts sagen.

**Reply by VirginEnt** [reply]
Und die richtigen CCU's würde man auch über "Hub hinzufügen" hinzufügen?

**Reply by Baxxy13** [reply]
Ja, kann man machen.
Es kommt dann eine Auswahl wo man entweder automatisch gefundene Instanzen oder manuell etwas hinzufügen kann.
Dann kommt man zu dem normalen Config-Flow.
